docs/rest_api/dynamic_content.rst

```diff
@@ -0,0 +1,430 @@
+     - int
+     - ID of the User that created the Dynamic Content
+   * - ``createdByUser``
+     - string
+     - Name of the User that created the Dynamic Content
+   * - ``dateModified``
+     - datetime/null
+     - Date/time Dynamic Content was last modified
+   * - ``modifiedBy``
+     - int
+     - ID of the User that last modified the Dynamic Content
+   * - ``modifiedByUser``
+     - string
+     - Name of the User that last modified the Dynamic Content
+   * - ``variantChildren``
+     - array
+     - Array of Dynamic Content entities for variants of this landing Dynamic Content
+   * - ``variantParent``
+     - object
+     - The parent/main Dynamic Content if this is a variant, also known as A/B test
+   * - ``sentCount``
+     - int
+     - Count of how many times the Dynamic Content got sent
+
+.. vale off
+
+List Dynamic Contents
+*********************
+
+.. vale on
+
+.. code-block:: php
+
+   <?php
+   // ...
+
+   $dynamicContents = $dynamicContentApi->getList($searchFilter, $start, $limit, $orderBy, $orderByDir, $publishedOnly, $minimal);
+
+.. vale off
+
+**HTTP Request**
+
+.. vale on
+
+``GET /dynamiccontents``
+
+**Query Parameters**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Description
+   * - ``search``
+     - String or search command to filter entities by.
+   * - ``start``
+     - Starting row for the entities returned. Defaults to 0.
+   * - ``limit``
+     - Limit number of entities to return. Defaults to the system configuration for pagination - defaults to 30.
+   * - ``orderBy``
+     - Column to sort by. Can use any column listed in the response.
+   * - ``orderByDir``
+     - Sort direction: ``asc`` or ``desc``.
+   * - ``publishedOnly``
+     - Only return currently published entities.
+   * - ``minimal``
+     - Return only array of entities without additional lists in it.
+
+
+**Response**
+
+``Expected Response Code: 200``
+
+.. code-block:: json
+
+   {
+       "total":30,
+       "dynamicContents":[
+           {
+               "isPublished":true,
+               "dateAdded":"2016-06-20T11:27:09+00:00",
+               "createdBy":1,
+               "createdByUser":"John Doe",
+               "dateModified":"2016-08-22T17:14:01+00:00",
+               "modifiedBy":1,
+               "modifiedByUser":"John Doe",
+               "id":2,
+               "name":"CD2",
+               "category":null,
+               "publishUp":null,
+               "publishDown":null,
+               "sentCount":0,
+               "variantParent":null,
+               "variantChildren":[]
+           }
+       ]
+   }
+
+**Properties**
+
+Same as `Get Dynamic Content <#get-dynamic-content>`_.
+
+.. vale off
+
+Create Dynamic Content
+**********************
+
+.. vale on
+
+.. code-block:: php
+
+   <?php
+
+   $data = array(
+       'name'        => 'Dynamic Content A',
+       'isPublished' => 1
+   );
+
+   $dynamicContent = $dynamicContentApi->create($data);
+
+Create a new dynamicContent.
```

Comment:
```suggestion
Create a new Dynamic Content.
```
I _think_ these should be separate words unless it's purposely written as above in the code sample.